plutarch-test/plutarch-base/Plutarch/ApiSpec.hs

```diff
@@ -202,3 +272,6 @@ getFields = phoistAcyclic $ plam $ \addr -> psndBuiltin #$ pasConstr # addr
 
 dummyCurrency :: CurrencySymbol
 dummyCurrency = Value.currencySymbol "\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11\x11"
+
+pshouldReallyBe :: ClosedTerm a -> ClosedTerm a -> Expectation
+pshouldReallyBe a b = pshouldBe b a
```

Comment:
Hmm, what is this for?

Reply:
Use of `pshouldBe` was giving me ambiguous type errors, so I decided to tighten the constraints. I also swapped the arguments to make it more convenient.